packages/patternfly-4/react-charts/package.json

```diff
@@ -41,10 +41,10 @@
     "develop": "yarn build:babel:esm --skip-initial-build --watch --verbose"
   },
   "optionalDependencies": {
-    "@types/victory": "^0.9.19",
+    "@types/victory": "^31.0.15",
     "hoist-non-react-statics": "^3.1.0",
```

Comment:
Can you please update hoist-non-react-statics? Cost Management will not run without this also being updated.

![Screen Shot 2019-05-01 at 11 02 05 AM](https://user-images.githubusercontent.com/17481322/57023654-99582000-6c00-11e9-97dc-6d6a0175a2d4.png)

Reply:
Updated